docs/get_cover_gallery.py

```diff
@@ -7,11 +7,70 @@
 style.use("preliz-doc")
 rng = np.random.default_rng(247)
 
+category_colors = {
+    'continuous': 'C3',
+    'discrete': 'C1',
+    'modifier': 'C2'
+}
+
+distribution_categories = {
```

Comment:
No need to do this manually. All distributions are subclasses of classes matching this color mapping.

Reply:
Working on it